pages/pil/dashboard/index.js

```diff
@@ -45,6 +46,8 @@ module.exports = settings => {
 
   app.use('/procedures', require('../procedures')());
 
+  app.use('/exemptions/modules', require('../modules_exempt')());
```

Comment:
Should this child route be added in `pages/pil/exemptions/index.js`?

Reply:
Nope,it needs to go before ```app.use('/exemptions', require('../exemptions')());``` because otherwise  _/exemptions_ will catch it and _/exemptions/modules_ will not be found.

Reply:
_hint:_ It should go in the child page

Reply:
change the order in the child page if it it being caught by the /